docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml

```diff
@@ -0,0 +1,356 @@
+              - - $%compilers
+            - matrix:
+              - - $mpi_packages
+              - - $%compilers
+              - - $^mpis
+      # Un-comment and update mirror_url to install from spack cache
+      # spack_cache_url:
+      # - mirror_name: gcs_cache
+      #   mirror_url: gs://...
+
+  - id: spack-builder-startup  # TODO: Update example or make generic
+    source: modules/scripts/startup-script
+    settings:
+      runners:
+      - $(spack.install_spack_deps_runner)
+      - $(spack.install_spack_runner)
+      - type: shell
+        destination: setup_gromacs.sh
+        content: |
+          #!/bin/bash
+          source /apps/spack/share/spack/setup-env.sh
+          spack env activate gromacs
+          chmod -R a+rwX /apps/spack/var/spack/environments/gromacs
+          mkdir -p /apps/gromacs
+          chmod a+rwx /apps/gromacs
+          cd /apps/gromacs
+          wget --no-verbose https://ftp.gromacs.org/pub/benchmarks/water_GMX50_bare.tar.gz
+          tar xzf water_GMX50_bare.tar.gz
+
+      - type: data  # rethink
+        destination: /apps/gromacs/submit_gromacs.sh
+        content: |
+          #!/bin/bash
+          #SBATCH -N 2
+          #SBATCH --ntasks-per-node 30
+          #SBATCH -p compute
+
+          # Size can be 0000.65  0000.96  0001.5  0003  0006  0012  0024  0048  0096  0192  0384  0768  1536  3072
+          # Type can be 'pme' or 'rf'
+
+          source /apps/spack/share/spack/setup-env.sh
+          spack env activate gromacs
+
+          # Check that gmx_mpi exists
+          which gmx_mpi
+          cd $SLURM_SUBMIT_DIR
+          cp /apps/gromacs/water-cut1.0_GMX50_bare/1536/* .
+          scontrol show hostnames ${SLURM_JOB_NODELIST} > hostfile
+          gmx_mpi grompp -f pme.mdp -c conf.gro -p topol.top -o input.tpr
+          mpirun -n 60 -hostfile hostfile -ppn 30 gmx_mpi mdrun -notunepme -dlb yes -v -resethway -noconfout -nsteps 4000 -s input.tpr
+
+      - type: shell  # TODO: consider making bool option
+        destination: shutdown.sh
+        content: |
+          #!/bin/bash
+          if [ ! -f /etc/block_auto_shutdown ]; then
+                  touch /etc/block_auto_shutdown
+                  shutdown -h +1
+          fi
+
+  - id: spack-builder
+    source: modules/compute/vm-instance
+    use: [network-install, appsfs-install, spack-builder-startup]
+    settings:
+      name_prefix: spack-builder
+      machine_type: c2-standard-16  # TODO Consider bumping up, left low for quota
+
+- group: cluster
+  modules:
+
+  - id: network1
+    source: modules/network/pre-existing-vpc
+
+  - id: homefs
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      server_ip: $(vars.homefs_server_ip)
+      remote_mount: nfsshare
+      local_mount: /home
+      fs_type: nfs
+
+  - id: appsfs
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      server_ip: $(vars.appsfs_server_ip)
+      remote_mount: nfsshare
```

Comment:
same comment about the remote_mount if you choose to do that.

Reply:
Done.